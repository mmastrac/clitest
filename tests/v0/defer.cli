#!/usr/bin/env clitest --v0

defer {
    $ rm -rf $TEMP_DIR
    *
}

$ mktemp -d
%SET TEMP_DIR
*
