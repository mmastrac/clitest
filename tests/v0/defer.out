Running <root>/defer.cli ...

mktemp -d
---
<tmp>
---
✅ OK

(deferred) rm -rf $TEMP_DIR
---
(no output)
---
✅ OK

<root>/defer.cli PASSED
